### ISSUE TITLE
How can I have 2 types of calendars?

### ISSUE THREAD

**Issue by Stamimail** [White-Tiger/T-Clock#26]
I want to have another Date in addition to the gregorian.

**Comment by White-Tiger** [comment]
That's not possible with T-Clock... you would have to find an external calendar that does what you need.... (that is, if such thing exists)

**Comment by Stamimail** [comment]
Do you aware that such a thing already exists bulid-in in Windows, for some languages?

**Comment by White-Tiger** [comment]
so Windows can display two calendars at once?
If so.. why are you not using it then?

**Comment by Stamimail** [comment]
>so Windows can display two calendars at once?
No. I wrote "I can't display 2 types simultaneously." This is the problem.
It letting me to choose only one type of calendar to display. The one or the other.
I only mentioned, that the other type is available built-in in (my) windows. I don't need to download/install it.
